features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml

```diff
@@ -1,20 +1,20 @@
 <!--
- ~ Copyright (c) 2005-2011, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
- ~
- ~ WSO2 Inc. licenses this file to you under the Apache License,
- ~ Version 2.0 (the "License"); you may not use this file except
- ~ in compliance with the License.
- ~ You may obtain a copy of the License at
- ~
- ~    http://www.apache.org/licenses/LICENSE-2.0
- ~
- ~ Unless required by applicable law or agreed to in writing,
- ~ software distributed under the License is distributed on an
- ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- ~ KIND, either express or implied.  See the License for the
- ~ specific language governing permissions and limitations
- ~ under the License.
- -->
+  ~ Copyright (c) 2005-2025, WSO2 LLC. (http://www.wso2.com).
```

Comment:
```suggestion
 ~ Copyright (c) 2005-2025, WSO2 LLC. (http://www.wso2.com).
```

Seems like unintended formatting

Reply:
Added license format is generated via the IDE and the correct formatting is to have two space indentation from the left corner.